cypress/integration/homePage.js

```diff
@@ -0,0 +1,210 @@
+import faker from "faker";
+
+import Customer from "../apiRequests/Customer";
+import { HOMEPAGE_SELECTORS } from "../elements/homePage/homePage-selectors";
+import HomePageSteps from "../steps/homePageSteps";
+import { urlList } from "../url/urlList";
+import ChannelsUtils from "../utils/channelsUtils";
+import HomePageUtils from "../utils/homePageUtils";
+import OrdersUtils from "../utils/ordersUtils";
+import ProductsUtils from "../utils/productsUtils";
+import ShippingUtils from "../utils/shippingUtils";
+
+// <reference types="cypress" />
+describe("User authorization", () => {
+  const startsWith = "Cy-";
+
+  const customer = new Customer();
+  const productsUtils = new ProductsUtils();
+  const shippingUtils = new ShippingUtils();
+  const ordersUtils = new OrdersUtils();
+  const channelsUtils = new ChannelsUtils();
+  const homePageUtils = new HomePageUtils();
+  const homePageSteps = new HomePageSteps();
+
+  let customerId;
+  let defaultChannel;
+  const productPrice = 22;
+  const shippingPrice = 12;
+  const randomName = startsWith + faker.random.number();
+  const randomEmail = randomName + "@example.com";
+
+  before(() => {
+    cy.clearSessionData().loginUserViaRequest();
+    productsUtils.deleteProperProducts(startsWith);
+    customer.deleteCustomers(startsWith);
+    shippingUtils.deleteShipping(startsWith);
+    let addresses;
+
+    channelsUtils
+      .getDefaultChannel()
+      .then(channel => {
```

Comment:
Let's avoid nested `then`s. Async and await should do

Reply:
Cypress doesn't recommended to use async/await commands. These commands don't behave as expected when running code on cypress.

Reply:
Ok, seen it - makes sense 👍